config/crd/cluster.open-cluster-management.io_hypershiftdeployments.yaml

```diff
@@ -40,6 +40,10 @@ spec:
       jsonPath: .status.conditions[?(@.type=="PlatformIAMConfigured")].status
       name: Ready
       type: string
+    - description: Reason
+      jsonPath: .status.conditions[?(@.type=="ManifestWorkConfigured")].reason
+      name: MANIFEST
```

Comment:
Maybe I should have called this MANIFESTWORK...

Reply:
I vote for manifestwork, manifest is a bit too generic and could be understand as the manifest of hypershiftDeployment or it's children resource

Reply:
yes, agree with Ian